contentgrid-appserver-application-model/src/main/java/com/contentgrid/appserver/application/model/searchfilters/FullTextSearchAttributeSearchFilter.java

```diff
@@ -0,0 +1,63 @@
+package com.contentgrid.appserver.application.model.searchfilters;
+
+import com.contentgrid.appserver.application.model.Application;
+import com.contentgrid.appserver.application.model.i18n.ConfigurableTranslatable;
+import com.contentgrid.appserver.application.model.i18n.TranslatableImpl;
+import com.contentgrid.appserver.application.model.searchfilters.flags.SearchFilterFlag;
+import com.contentgrid.appserver.application.model.values.FilterName;
+import com.contentgrid.appserver.application.model.values.PropertyPath;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Singular;
+
+import java.util.Locale;
+import java.util.Set;
+
+import static com.contentgrid.appserver.application.model.searchfilters.AttributeSearchFilter.Operation.FTS;
+
+/**
+ * FullTextSearchAttributeSearchFilter is a search filter that performs full-text search operations on a specified attribute.
+ * <br>
+ * The main difference between this and a regular {@link AttributeSearchFilter} is that this filter specifies a {@link Locale}.
+ */
+public class FullTextSearchAttributeSearchFilter extends AttributeSearchFilter implements LocaleAwareSearchFilter {
+
+    /**
+     * The locale for which the search filter is defined.
+     * Might be null, in which case the default locale for the {@link com.contentgrid.appserver.application.model.Application} will be used.
+     */
+    Locale locale;
+
+    public Locale getLocale(@NonNull Application application) {
+        return locale == null? application.getLocale() : locale;
+    }
```

Comment:
1. I think we should not have any fallback-to-defaults at the configuration level. We can have configuration settings that are global for the application, but the configuration generation will just copy that to all the necessary fields instead of having a global setting.
2. This API is _weird_: needing to pass in the application object where we got the search filter from is weird